partner-groups/adding-or-updating-groups.md

```diff
@@ -0,0 +1,16 @@
+## Updating an existing group
+We welcome PRs regarding updates to an existing group when the name of the group, the link to group or the charter of the group changes or if the group is retired
+
+## Adding a new group
+
+We welcome PRs regarding addition of new groups, when:
+1. The group is not part of the existing list
+2. The group is relevant to security
+3. Ownership of the group is shared with more than one independent entities
+4. Information such as charter, projects, artifacts of the group should be publicly accessible
+
+## Naming convention
+
+1. Alphabetical ordering is preferred
```

Comment:
let's `--Wall` to make this enforceable :) 
```suggestion
1. Groups named in alphabetical order
```

Reply:
```suggestion
1. Groups names in alphabetical order by type
```